### PR TITLE
[Feature] [connector] Support Spark Batch Source/Sink of Datalake delta 

### PR DESCRIPTION
**Pull request by huzk8**
## Purpose of this pull request

this pull request is related to #1031  #1032 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Review by RickyHuo — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please add apache license header firstly :)

File: docs/en/spark/configuration/sink-plugins/Delta.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Write Rows to  Delta Lake.
+
+## Options
+
+| name | type | required | default value | engine |
+| --- | --- | --- | --- | --- |
+| delta.sink.path | string | yes | - | Spark |
+| save_mode	 | string | no | append | Spark |
+
+[More Delta](https://docs.delta.io/0.6.0/delta-batch.html#write-to-a-table)
```
Comment (an-shi-chi-fan) [inline docs/en/spark/configuration/sink-plugins/Delta.md:14]: is ok to ref other link ? @CalvinKirs

**Review by RickyHuo — CHANGES_REQUESTED** [review]
According to https://docs.delta.io/0.6.0/delta-batch.html#table-batch-reads-and-writes.

replaceWhere seems like a necessary option for sink.

versionAsOf and timestampAsOf are optional config for source.

What do you think of these options ?

**Comment by Kyofin** [comment]
@CalvinKirs @RickyHuo  review request.

**Comment by CalvinKirs** [comment]
no updates, if you have any updates, you can leave a message, and I will reopen it.

**Pull request closed without merging**